dev-lang/rust/rust-1.23.0-r1.ebuild

```diff
@@ -19,15 +19,17 @@ else
 	SLOT="stable/${ABI_VER}"
 	MY_P="rustc-${PV}"
 	SRC="${MY_P}-src.tar.gz"
-	KEYWORDS="~amd64 ~x86"
+	KEYWORDS="~amd64 ~x86 ~arm64"
```

Comment:
In alphabetical order please. Also, are you sure this is not compatible with 32 bit arm?